src/cartesian/BrushY.tsx

```diff
@@ -0,0 +1,570 @@
+/**
```

Comment:
I didn't even see this file... I wonder if we can do these changes without a new component and instead with an `orientation` prop or similar

Reply:
The problem is not being able to render the same component twice. If that's solved, than BrushY could be integrated into Brush

Reply:
The pattern to solve this is done for the Axis. A shared CartesianAxis exists, but then we also have XAxis and YAxis. Both of them internally use CartesianAxis.